x-pack/test/encrypted_saved_objects_api_integration/tests/encrypted_saved_objects_api.ts

```diff
@@ -361,6 +385,32 @@ export default function ({ getService }: FtrProviderContext) {
       expect(rawAttributes.privateProperty).to.not.be(updatedAttributes.privateProperty);
     });
 
+    it('#update overwrites objects in their entirety', async () => {
```

Comment:
__question:__ would you mind adding one more test here that tests the scenario with removed fields that actually forced us to do that change (e.g one top level and one nested)? I know you're testing this in WebHook test, but having this in ESO tests would also be beneficial.

Reply:
discussed synchronously - this is already covered 👍